### ISSUE TITLE
Fix: id entry in databases can be duplicated (kegg.tsv and uniprot.tsv)

### ISSUE THREAD

**Issue by ae-tafur** [SysBioChalmers/GECKO#315]
### Description of the bug:

Recently, working in a non-model organism I notice that it have two genes (same sequence) with a different `Gene Name` but same `Entry (protein id)` . This creates the same reaction duplicated (usage_prot_*) as well as the reaction where the protein is involved. 

I proposed that `loadDatabases`must validate for unique `Gene Name` and `Entry`. In case it found duplicated entry, create a warning for the user. So the user can decide how to proceed.

**I hereby confirm that I have:**
- [ ] Tested my code with [all requirements](https://github.com/SysBioChalmers/GECKO) for running GECKO
- [ ] Done this analysis in the `main` branch of the repository
- [ ] Checked that a similar issue does not exist already

**Comment by edkerk** [comment]
Resolved in #349 for `uniprot.tsv`.

A similar implementation will not be made for `kegg.tsv`, as there it is not uncommon to have gathered a correct species-specific KEGG database and still have duplicate IDs.

For instance for _S. cerevisiae_: [sce:YNL030W](https://www.genome.jp/dbget-bin/www_bget?sce:YNL030W) and [sce:YBR009C](https://www.genome.jp/dbget-bin/www_bget?sce:YBR009C) both refer to [P02309](https://www.uniprot.org/uniprotkb/P02309/entry).

At the same time, the `uniprot.tsv` would have just one `P02309` that is instead assigned the _Gene Name_ `YBR009C; YNL030W`. One can argue that _this_ is actually problematic, but _if_ it is conflicting with the model reconstruction it would actually be reported as a gene in `noUniprot` when running `makeEcModel`. In contrast, the problem mentioned by @ae-tafur would go unnoticed. So, "duplications" like these should not throw an error.

Going back to KEGG then, having duplicate Uniprot IDs is also not detrimental, KEGG database is only used for extracting EC numbers, and not to define usage_prot_* reactions as mentioned above.

tl:dr; with #349, this issue is resolved.
